### DISCUSSION TITLE
Are the packages published somewhere?

### DISCUSSION THREAD

**Discussion by jeanplevesque** [AdamEssenmacher/GoogleApisForiOSComponents#3] [Q&A] [answered]
Hi!

I see that your project files are updated to the 10.16.0 SDK version and I'd like to leverage that in my mobile project. However I don't find packages that point back to this repo on nuget.org so I'm wondering if they are (or will be) published?

Thanks!

**Comment by giuseppenovielli** [comment, marked as answer]
https://www.nuget.org/packages?q=AdamE.Firebase.iOS&includeComputedFrameworks=true&prerel=true&sortby=relevance

**Reply by jeanplevesque** [reply]
Thanks!
